app/_jekyll_plugins/versions.rb

```diff
@@ -11,7 +11,7 @@ def generate(site)
       # Add a `version` property to every versioned page
       site.pages.each do |page|
         parts = Pathname(page.path).each_filename.to_a
-        if parts[0] == site.config["documentation"]
+        if parts[0] == "docs"
```

Comment:
when I actually removed the variable from the `yml` file it broke, not exactly sure why, so I brought it back there, but forgot about this spot.